datafusion/proto/src/generated/datafusion_proto_common.rs

```diff
@@ -722,19 +722,19 @@ pub mod column_options {
         StatisticsEnabled(::prost::alloc::string::String),
     }
     #[allow(clippy::derive_partial_eq_without_eq)]
-    #[derive(Clone, PartialEq, ::prost::Oneof)]
+    #[derive(Clone, Copy, PartialEq, ::prost::Oneof)]
     pub enum BloomFilterFppOpt {
         #[prost(double, tag = "6")]
         BloomFilterFpp(f64),
     }
     #[allow(clippy::derive_partial_eq_without_eq)]
-    #[derive(Clone, PartialEq, ::prost::Oneof)]
+    #[derive(Clone, Copy, PartialEq, ::prost::Oneof)]
```

Comment:
just wondering why the Copy was added?

Reply:
This is  generated code. so I think it comes from the changes in prost. I think it comes from https://github.com/tokio-rs/prost/pull/950 specifically